tests/bud.bats

```diff
@@ -36,7 +36,7 @@ load helpers
   for var in "${checkvars[@]}"; do
     echo "ARG TARGET${var}" >>$containerfile
   done
-  echo "COPY . ." >>$containerfile
+  echo "RUN echo hi" >>$containerfile
```

Comment:
Given that this is built explicitly for `linux/amd64/v2` below, won't this fail on other architectures?

Reply:
So, that confused me when I first looked at this. Doesn't buildah automatically use the Containerfile directory as its context directory? Shouldn't `COPY . .` run from that empty directory? I tried changing `-f $containerfile` to `${TEST_SCRATCH_DIR}` but podman still fails. This seems like a major misunderstanding on my part.

Reply:
I would expect `COPY . .` to pick up the Containerfile that's just been written there.  How does it fail?

Reply:
Problem found. Fix in progress. Totally my fault.

Reply:
Ah yes, `ENV` or `LABEL` is better here, Ed has already created a PR for that here: https://github.com/containers/buildah/pull/4320